src/flags.ts

```diff
@@ -9,22 +9,22 @@ type NotArray<T> = T extends Array<any> ? never: T;
 
 export function custom<T = string, P extends CustomOptions = CustomOptions>(
   defaults: Partial<OptionFlag<T[], P>> & {
-    multiple: true
+    multiple: true;
```

Comment:
semicolons are required for types?

Reply:
Not required. I just added it for consistency